### ISSUE TITLE
React 16

### ISSUE THREAD

**Issue by GuillaumeCisco** [gaearon/redux-devtools-log-monitor#69]
Hey there.
Does this tool work with react 16?
I have issues like :

```
warning.js:33 Warning: Expected `onClick` listener to be a function, instead got a value of `boolean` type.
    in label (created by JSONNestedNode)
    in li (created by JSONNestedNode)
    in JSONNestedNode (created by JSONObjectNode)
    in JSONObjectNode (created by JSONNode)
    in JSONNode (created by JSONNestedNode)
    in ul (created by JSONNestedNode)
    in li (created by JSONNestedNode)
    in JSONNestedNode (created by JSONObjectNode)
    in JSONObjectNode (created by JSONNode)
    in JSONNode (created by JSONTree)
    in ul (created by JSONTree)
    in JSONTree (created by LogMonitorEntry)
    in div (created by LogMonitorEntry)
    in div (created by LogMonitorEntry)
    in LogMonitorEntry (created by LogMonitorEntryList)
    in div (created by LogMonitorEntryList)
    in LogMonitorEntryList (created by LogMonitor)
    in div (created by LogMonitor)
    in div (created by LogMonitor)
    in LogMonitor
    in FilterMonitor
    in div (created by Dock)
    in div (created by Dock)
    in div (created by Dock)
    in Dock (created by DockMonitor)
    in DockMonitor (created by Connect(DockMonitor))
    in Connect(DockMonitor) (created by DevTools)
    in DevTools (created by Root)
    in div (created by Root)
    in MuiThemeProvider (created by Root)
    in Provider (created by Root)
    in Root
    in AppContainer
```

**Comment by GuillaumeCisco** [comment]
Ok just found the problem.
It is due to the dependency react-json-tree.
I will make a PR. The issue is here : https://github.com/alexkuz/react-json-tree/blob/master/src/JSONNestedNode.js#L169

**Comment by GuillaumeCisco** [comment]
Just found an issue on the react-json-tree repo : https://github.com/alexkuz/react-json-tree/issues/94
Added the PR too.
Hope it will be deployed fast.
Cheers

**Comment by jadir-junior** [comment]
this issue was solve? I have the same problem!

**Comment by GuillaumeCisco** [comment]
Nope, it was one month ago.
No hope, please upvote the issue on react json tree

**Comment by GuillaumeCisco** [comment]
Just created a PR for updateing redux-devtools-log-monitor : https://github.com/gaearon/redux-devtools-log-monitor/pull/71

**Comment by abuelwafa** [comment]
@GuillaumeCisco I had created the same PR #70 few hours before you. hopefully anyone of them gets merged and released soon.

**Comment by GuillaumeCisco** [comment]
Thanks @abuelwafa, sorry I did not see it. My bad

**Comment by joyfulelement** [comment]
Had the same issue with the recent upgrad to react 16. If anyone needs this fix _now_ before https://github.com/gaearon/redux-devtools-log-monitor/pull/70 is merged, there is a temporary workaround for this.

## Workaround with version `1.3.0` 
While waiting for `redux-devtools-log-monitor` library to upgrade its `react-json-tree` dependency to `0.11.0` (https://github.com/gaearon/redux-devtools-log-monitor/pull/70), in the meantime, for people we use `yarn` for dependency management could take advantage of the [selective version resolution feature](https://yarnpkg.com/lang/en/docs/selective-version-resolutions/) introduced in Yarn 1.x to combat this incompatibility issue. 

Simply have the following lines to your `package.json` file and run `yarn install`.

```
"devDependencies": {
    ...
    "redux-devtools-log-monitor": "1.3.0",
    ...
},
"resolutions": {
    "redux-devtools-log-monitor/react-json-tree": "0.11.0"
}

```

So far with my local testing, the log monitor displays fine with no warning on Chrome (v63.0.3239.18 (Official Build) beta (64-bit)) console. I hope this information is helpful for anyone experienced the same issue.

**Comment by GuillaumeCisco** [comment]
Sorry to highlight you @gaearon, I know you have a lot of stuff to do.
But is this possible to merge thix fix and deploy to npm asap?

Thank you very much

**Comment by zalmoxisus** [comment]
Sorry for the delay, guys. I published it as `redux-devtools-log-monitor@1.4.0`.  Also make sure to update `redux-devtools` to `3.4.1` to have React 16 as a peer dependency.
